PrebidMobile/PrebidMobileCore/src/main/java/org/prebid/mobile/core/CacheManager.java

```diff
@@ -0,0 +1,153 @@
+package org.prebid.mobile.core;
+
+import android.content.Context;
+import android.os.Handler;
+import android.os.Looper;
+import android.text.TextUtils;
+import android.webkit.WebView;
+
+import org.apache.commons.lang3.RandomStringUtils;
+import org.apache.commons.lang3.StringEscapeUtils;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class CacheManager {
+    // This is the class that manages three cache instances, one for DFP, one for MoPub and one for SDK rendered ad
+    private WebView dfpWebCache;
+    private WebView mopubWebCache;
```

Comment:
We don't need mopub related web cache code here anymore right? since we're using server side cache.

Reply:
Yes I'll remove this.